File: 16 - Legal Driving Age/Golang/main.go
// Package main provides ...
package main

import (
    "fmt"
)

func main() {
    var age int
    fmt.Print("What is your age? ")
    fmt.Scanf("%d", &age)
    if age >= 16 {
        fmt.Println("You are old enough to legally drive.")
    } else {
        fmt.Println("You are not old enough to legally drive.")
    }
}

File: 16 - Legal Driving Age/C++/main.cpp
#include <iostream>

using namespace std;

int main(void)
{
    int age = 0;

    cout << "What is your age? ";
    cin >> age;

    if (age >= 16) {
        cout << "You are old enough to legally drive." << endl;
        return 0;
    } else {
        cout << "You are not old enough to legally drive." << endl;
        return 0;
    }
}
